README.md

```diff
@@ -9,14 +9,19 @@
 ## Questions
 
 1.  What is React JS and what problems does it try and solve?
+React is a user-interface library that manages dynamic data and put it on the front-end of an app. It helps with changes on the app because of user interaction.
```

Comment:
It saves resources/time by changing a virtual dom and comparing it to the actual DOM before requesting changes.